docs/install/install.rst

```diff
@@ -0,0 +1,318 @@
+            ``hipamd/src``.
+
+            Usage:
+
+            .. code:: shell
+
+               `hip_prof_gen.py [-v] <input HIP API .h file> <patched srcs path> <previous output> [<output>]`
+
+            Flags:
+
+               * ``-v``: Verbose messages
+               * ``-r``: Process source directory recursively
+               * ``-t``: API types matching check
+               * ``--priv``: Private API check
+               * ``-e``: On error exit mode
+               * ``-p``: ``HIP_INIT_API`` macro patching mode
+
+            Example usage:
+
+            .. code:: shell
+
+               hip_prof_gen.py -v -p -t --priv <hip>/include/hip/hip_runtime_api.h \
+               <hipamd>/src <hipamd>/include/hip/amd_detail/hip_prof_str.h \
+               <hipamd>/include/hip/amd_detail/hip_prof_str.h.new
+
+   .. tab-item:: NVIDIA
+      :sync: nvidia
+
+      #. Get the HIP source code.
+
+         .. code:: shell
+
+            git clone -b "$ROCM_BRANCH" https://github.com/ROCm-Developer-Tools/hip.git
+            git clone -b "$ROCM_BRANCH" https://github.com/ROCm-Developer-Tools/clr.git
+            git clone -b "$ROCM_BRANCH" https://github.com/ROCm-Developer-Tools/HIPCC.git
+
+      #. Set the environment variables.
+
+         .. code:: shell
+
+            export HIP_DIR="$(readlink -f hip)"
+            export CLR_DIR="$(readlink -f hipamd)"
+            export HIPCC_DIR="$(readlink -f hipcc)"
+
+      #. Build the HIPCC runtime.
+
+         .. code:: shell
+
+            cd "$HIPCC_DIR"
+            mkdir -p build; cd build
+            cmake ..
+            make -j4
+
+      #. Build HIP.
+
+         .. code:: shell
+
+            cd "$CLR_DIR"
+            mkdir -p build; cd build
+            cmake -DHIP_COMMON_DIR=$HIP_DIR -DHIP_PLATFORM=nvidia -DCMAKE_INSTALL_PREFIX=$PWD/install -DHIPCC_BIN_DIR=$HIPCC_DIR/build -DHIP_CATCH_TEST=0 -DCLR_BUILD_HIP=ON -DCLR_BUILD_OCL=OFF ..
+            make -j$(nproc)
+            sudo make install
+
+Build HIP tests
+=================================================
+
+.. tab-set::
+
+   .. tab-item:: AMD
+      :sync: amd
+
+      * Build HIP directed tests.
+
+         .. code:: shell
+
+            sudo make install
+            make -j$(nproc) build_tests
+
+         By default, all HIP directed tests are built and generated in
+         ``$CLR_DIR/build/hipamd/directed_tests``.
+
+         * Run all HIP ``directed_tests``.
+
+            .. code:: shell
+
+               ctest
+
+            or
+
+            .. code:: shell
+
+               make test
+
+
+         * Build and run a single directed test.
+
+            .. code:: shell
+
+               make directed_tests.texture.hipTexObjPitch
+               cd $CLR_DIR/build/hipamd/directed_tests/texture
+               ./hipTexObjPitch
+
+         .. note::
+            The integrated HIP directed tests will be deprecated in a future release.
+
+      * Build HIP catch tests.
```

Comment:
We should not call it catch tests, talk to HIP team. They are probably unit tests, functional tests, etc.